src/Microsoft.Health.Dicom.Core/Features/Workitem/IWorkitemDatasetValidator.cs

```diff
@@ -16,13 +16,10 @@ public interface IWorkitemDatasetValidator
         string Name { get; }
 
         /// <summary>
-        /// Validates the <paramref name="dicomDataset"/>.
+        /// Validates the <paramref name="dataset"/>.
         /// </summary>
-        /// <param name="dicomDataset">The DICOM dataset to validate.</param>
-        /// <param name="workitemInstanceUid">
-        /// If supplied, the workitemInstanceUid in the <paramref name="dicomDataset"/> must match to be considered valid.
-        /// </param>
+        /// <param name="dataset">The DICOM dataset to validate.</param>
         /// <exception cref="DatasetValidationException">Thrown when the validation fails.</exception>
-        void Validate(DicomDataset dicomDataset, string workitemInstanceUid);
+        void Validate(DicomDataset dataset);
```

Comment:
The query param will already be reconciled with the SopInstanceUID attribute in the dataset.